pacehrh/R/pace_coverage_config.R

```diff
@@ -0,0 +1,78 @@
+#' Load CoverageRates Values
+#'
+#' Read the annual task coverage values from the model inputs Excel file. The
+#' name and location of the model inputs Excel file is loaded from the global
+#' configuration JSON file.
+#'
+#' @param sheetName Sheet name from the model input Excel file
+#'
+#' @return Data table of per-year task coverage levels
+#'
+loadCoverageRates <- function(sheetName = .defaultCoverageRatesSheet){
+  traceMessage(paste0("Loading task coverage rates sheet ", sheetName))
+  
+  coverageRatesData <- NULL
```

Comment:
This line isn't necessary. (Immediately overwritten by loadTable() return value.)